sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/documentanalysis/package-info.java

```diff
@@ -2,8 +2,297 @@
+ *         DocumentModelBuildMode.TEMPLATE,
+ *         prefix,
+ *         new BuildDocumentModelOptions&#40;&#41;.setModelId&#40;&quot;my-custom-built-model&quot;&#41;.setDescription&#40;&quot;model desc&quot;&#41;,
+ *         Context.NONE&#41;;
+ *
+ * DocumentModelDetails customBuildModel = buildOperationPoller.getFinalResult&#40;&#41;;
+ *
+ * &#47;&#47; analyze using custom-built model
+ * String modelId = customBuildModel.getModelId&#40;&#41;;
+ * String documentUrl = &quot;documentUrl&quot;;
+ * SyncPoller&lt;OperationResult, AnalyzeResult&gt; analyzeDocumentPoller =
+ *     documentAnalysisClient.beginAnalyzeDocumentFromUrl&#40;modelId, documentUrl&#41;;
+ *
+ * AnalyzeResult analyzeResult = analyzeDocumentPoller.getFinalResult&#40;&#41;;
+ *
+ * for &#40;int i = 0; i &lt; analyzeResult.getDocuments&#40;&#41;.size&#40;&#41;; i++&#41; &#123;
+ *     final AnalyzedDocument analyzedDocument = analyzeResult.getDocuments&#40;&#41;.get&#40;i&#41;;
+ *     System.out.printf&#40;&quot;----------- Analyzing custom document %d -----------%n&quot;, i&#41;;
+ *     System.out.printf&#40;&quot;Analyzed document has doc type %s with confidence : %.2f%n&quot;,
+ *         analyzedDocument.getDocType&#40;&#41;, analyzedDocument.getConfidence&#40;&#41;&#41;;
+ *     analyzedDocument.getFields&#40;&#41;.forEach&#40;&#40;key, documentField&#41; -&gt; &#123;
+ *         System.out.printf&#40;&quot;Document Field content: %s%n&quot;, documentField.getContent&#40;&#41;&#41;;
+ *         System.out.printf&#40;&quot;Document Field confidence: %.2f%n&quot;, documentField.getConfidence&#40;&#41;&#41;;
+ *         System.out.printf&#40;&quot;Document Field Type: %s%n&quot;, documentField.getType&#40;&#41;&#41;;
+ *         System.out.printf&#40;&quot;Document Field found within bounding region: %s%n&quot;,
+ *             documentField.getBoundingRegions&#40;&#41;.toString&#40;&#41;&#41;;
+ *     &#125;&#41;;
+ * &#125;
+ *
+ * analyzeResult.getPages&#40;&#41;.forEach&#40;documentPage -&gt; &#123;
+ *     System.out.printf&#40;&quot;Page has width: %.2f and height: %.2f, measured with unit: %s%n&quot;,
+ *         documentPage.getWidth&#40;&#41;,
+ *         documentPage.getHeight&#40;&#41;,
+ *         documentPage.getUnit&#40;&#41;&#41;;
+ *
+ *     &#47;&#47; lines
+ *     documentPage.getLines&#40;&#41;.forEach&#40;documentLine -&gt;
+ *         System.out.printf&#40;&quot;Line '%s' is within a bounding box %s.%n&quot;,
+ *             documentLine.getContent&#40;&#41;,
+ *             documentLine.getBoundingPolygon&#40;&#41;.toString&#40;&#41;&#41;&#41;;
+ *
+ *     &#47;&#47; words
+ *     documentPage.getWords&#40;&#41;.forEach&#40;documentWord -&gt;
+ *         System.out.printf&#40;&quot;Word '%s' has a confidence score of %.2f.%n&quot;,
+ *             documentWord.getContent&#40;&#41;,
+ *             documentWord.getConfidence&#40;&#41;&#41;&#41;;
+ * &#125;&#41;;
+ *
+ * &#47;&#47; tables
```

Comment:
The comments in code samples can be added/improved to be more descriptive.
Applies to all code samples